backend/de.metas.deliveryplanning.base/src/main/sql/postgresql/system/43-de.metas.deliveryplanning/5674860_sys_gh14481_Export_Delivery_Planning_Lines_process.sql

```diff
@@ -0,0 +1,85 @@
+-- Value: Delivery Planning Export (Excel)
+-- Classname: de.metas.report.jasper.client.process.JasperReportStarter
+-- JasperReport: @PREFIX@/de/metas/docs/deliveryplanning/deliveryplanning.xls
+-- 2023-02-01T16:18:42.371Z
+INSERT INTO AD_Process (AccessLevel,AD_Client_ID,AD_Org_ID,AD_Process_ID,AllowProcessReRun,Classname,CopyFromProcess,Created,CreatedBy,EntityType,IsActive,IsApplySecuritySettings,IsBetaFunctionality,IsDirectPrint,IsFormatExcelFile,IsNotifyUserAfterExecution,IsOneInstanceOnly,IsReport,IsTranslateExcelHeaders,IsUpdateExportDate,IsUseBPartnerLanguage,JasperReport,LockWaitTimeout,Name,PostgrestResponseFormat,RefreshAllAfterExecution,ShowHelp,SpreadsheetFormat,SQLStatement,Type,Updated,UpdatedBy,Value) VALUES ('3',0,0,585207,'Y','de.metas.report.jasper.client.process.JasperReportStarter','N',TO_TIMESTAMP('2023-02-01 18:18:42','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','N','N','Y','Y','N','N','Y','Y','N','Y','@PREFIX@/de/metas/docs/deliveryplanning/deliveryplanning.xls',0,'Delivery Planning Export Excel','json','N','N','xls','
+SELECT div.documentno,
```

Comment:
this select is a bit unwieldy for the application dictionary - please extract it into a view

Reply:
@metas-ts not sure is possible because it contains that "SQL where clause"  context variable....

Reply:
@teosarca @metas-ts 
I can do it by including the M_Delivery_Planning_ID column into the view and writing smth like:
`SELECT * FROM that_view AS M_Delivery_Planning WHERE @SELECTION_WHERECLAUSE/false@`
Sintacticaly it should work, however it feels more of a hack as a plain SQL.

Reply:
hm..ok, fine then..feel free to stay with the full select

Reply:
tested, works with the view and aforementioned SQL.